src/screens/shared/farmProfileEdit/FarmProfileEditForm.js

```diff
@@ -22,6 +30,20 @@ const styles = {
   }
 };
 class FarmProfileEditForm extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      farm: {},
```

Comment:
We shouldnt create state here in the form. Use formValues given by props and use onChange to update the formValues object.

Reply:
resolved.